objcgen/classhelper.cs

```diff
@@ -25,6 +25,7 @@ public ClassHelper (SourceWriter headers, SourceWriter implementation)
 		public string Name { get; set; }
 		public string Namespace { get; set; }
 		public string ManagedName { get; set; }
+		public string OuterClass { get; set; }
```

Comment:
I'll try it out and see what happens if I include the nested name in managedname